test/quantization/fx/test_quantize_fx.py

```diff
@@ -2658,7 +2659,7 @@ def forward(self, x):
             example_inputs = (torch.randn(3, 3),)
             # check prepared model
             m = prepare_fx(
-                original_m,
+                copy.deepcopy(original_m),
```

Comment:
To confirm -- `prepare_fx` is an inplace transformation?

Reply:
yes, both `prepare_fx` and `convert_fx` are inplace